components/layout_2020/geom.rs

```diff
@@ -222,42 +167,33 @@ impl fmt::Debug for flow_relative::Rect<Length> {
     }
 }
 
-impl<T: Clone> flow_relative::Vec2<T> {
-    pub fn size_to_physical(&self, mode: WritingMode) -> physical::Vec2<T> {
+impl<T: Clone + Copy> flow_relative::Vec2<T> {
```

Comment:
Why the `Copy` bound? Please either remove it, or remove the `Clone` bound and the calls to `.clone()` and the `&` expressions taking references to fields of `self`.

Reply:
Nice catch. Not sure how the `Copy` slipped in. I've fixed this.